src/main/java/htsjdk/variant/variantcontext/VariantContextBuilder.java

```diff
@@ -393,6 +452,7 @@ public VariantContextBuilder unfiltered() {
     public VariantContextBuilder genotypes(final GenotypesContext genotypes) {
         this.genotypes = genotypes;
         if ( genotypes != null )
```

Comment:
Here's a bug! Introduced by the lack of `{` `}`!

Please add some `{` `}`s.

Reply:
it wouldn't be that bad to validate a null genotypeContext...but sure, not as intended.